src/controllers/users.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+let users = [];
+
+const getAllUsers = (req, res) => {
+  if (res) {
+    res.send(users);
+  }
+
+  return users;
+};
+
+const getOne = (req, res) => {
+  const { userId } = req.params;
+  const foundUser = users.find(user => user.id === +userId);
+
+  if (!foundUser) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(foundUser);
+};
+
+const add = (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const newUser = {
+    id: Math.random(),
```

Comment:
Consider using a different method to generate unique ids, such as a counter or a library like uuid.

Reply:
Or make a function like `getNextId` where you find max id & plus 1

Reply:
It's not main, but your code should be clean & with best practice
Make a function for it, try to handle it!